### PR TITLE
swift-api-digester: Ignore the declarations that have no accessibility set. rdar://35375615

### PR DESCRIPTION
**Pull request by nkcsgexi**

### CODE REVIEW

**Comment by nkcsgexi** [comment]
@swift-ci please smoke test

**Comment by slavapestov** [comment]
@nkcsgexi I think this is worth investigating. Wrong accessibility can cause problems for code generation too. Do you mind tracking down which decls are affected?

**Comment by nkcsgexi** [comment]
@slavapestov you're right. This may indicate an importer issuer. I'm asking Jordan to provide some hints about what's going on here.

File: tools/swift-api-digester/swift-api-digester.cpp [hunk 1]
```diff
@@ -1282,6 +1282,9 @@ static bool shouldIgnore(Decl *D) {
       return true;
     if (VD->getBaseName().empty())
       return true;
+    // This shouldn't happen, being forgiving here.
+    if (!VD->hasAccess())
+      return true;
```
Comment (slavapestov) [inline tools/swift-api-digester/swift-api-digester.cpp]: I suggest adding a VD->dump() call here and running your test, then work backwards from there

Comment (nkcsgexi) [reply tools/swift-api-digester/swift-api-digester.cpp]: I have limited bandwidth for now. I'm not merging this.

Comment (slavapestov) [reply tools/swift-api-digester/swift-api-digester.cpp]: I'm not opposed to merging this, I just think you or @jrose-apple should investigate the underlying problem eventually :)

**Comment by jrose-apple** [comment]
It's in my queue, but it's not at the top. If we need to get unblocked fast then go ahead and merge this, but I'd appreciate a FIXME or even a follow-up Radar to take it back out.

**Pull request merged**